libs/@guardian/atoms-rendering/src/YoutubeAtom.stories.tsx

```diff
@@ -1,4 +1,4 @@
-import { ArticlePillar } from '@guardian/libs';
```

Comment:
ArticlePillar is deprecated so I've replaced with Pillar as per the recommendation